lightning/src/onion_message/messenger.rs

```diff
@@ -456,7 +485,7 @@ where
 	/// Handle an incoming onion message. Currently, if a message was destined for us we will log, but
 	/// soon we'll delegate the onion message to a handler that can generate invoices or send
 	/// payments.
```

Comment:
This is outdated. It can probably be removed since docs on trait implementations are fairly hidden